tests/aws/services/secretsmanager/test_secretsmanager.validation.json

```diff
@@ -101,14 +101,23 @@
   "tests/aws/services/secretsmanager/test_secretsmanager.py::TestSecretsManager::test_rotate_secret_invalid_lambda_arn": {
     "last_validated_date": "2024-03-15T10:11:13+00:00"
   },
+  "tests/aws/services/secretsmanager/test_secretsmanager.py::TestSecretsManager::test_rotate_secret_multiple_times_with_lambda_success[False]": {
+    "last_validated_date": "2025-03-16T07:42:50+00:00"
+  },
+  "tests/aws/services/secretsmanager/test_secretsmanager.py::TestSecretsManager::test_rotate_secret_multiple_times_with_lambda_success[True]": {
+    "last_validated_date": "2025-03-16T07:43:08+00:00"
+  },
   "tests/aws/services/secretsmanager/test_secretsmanager.py::TestSecretsManager::test_rotate_secret_with_lambda_success": {
     "last_validated_date": "2024-03-15T08:12:22+00:00"
   },
+  "tests/aws/services/secretsmanager/test_secretsmanager.py::TestSecretsManager::test_rotate_secret_with_lambda_success[False]": {
```

Comment:
As per my previous comment, can we remove this validation if it is not a currently tracked test combination?

Reply:
Same as I mentioned before, you are right. This should be `None` not False. I will update this